Backpack/src/androidTest/java/net/skyscanner/backpack/text/BpkTextTest.kt

```diff
@@ -182,8 +148,16 @@ class BpkTextTest {
 
       val font = getFont.invoke(withCustomFont)
 
-      Assert.assertTrue(message, font.isCustomFont)
-      Assert.assertEquals(message, ResourcesCompat.getFont(withCustomFont, net.skyscanner.backpack.test.R.font.shadows_into_light), font.typeface)
+      val expectedFont = when {
+        message.contains("Emphasized") ->
+          Typeface.create("cursive", Typeface.NORMAL)
+        message.contains("Heavy") ->
+          Typeface.create("casual", Typeface.NORMAL)
+        else ->
+          ResourcesCompat.getFont(withCustomFont, net.skyscanner.backpack.test.R.font.shadows_into_light)
```

Comment:
Can we do full import here?

Reply:
We already have another R import here, I'll add an alias